.github/workflows/release-changes.yml

```diff
@@ -33,16 +33,16 @@ jobs:
     - name: Updating the Release URL in POM 
```

Comment:
This workflow is.wrong please update with the latest one

Reply:
updated